app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,7 @@
 <resources>
-    <string name="app_name">Weatherapp</string>
+    <string name="app_name">My Weather App</string>
+    <string name="app_location">St.Paul, MN</string>
+    <string name="app_location_temperature">72°</string>
+    <string name="location_feel_like">Feel like 78°</string>
+    <string name="state_factors">Low 65°\nHigh 80°\nHumidity 100%\nPressure 1023hPa</string>
```

Comment:
While this way of handling the different lines is acceptable for this assignment it will become problematic as we drive the current conditions screen with data from the API. I suggest you split the view into four separate views before you merge and work on Assignment3.